src/display.c

```diff
@@ -93,24 +103,71 @@ static void OnDestroySurfaceRecord(void *data)
 	SurfaceRecord record = data;
 	Surface_Close(record->surface);
 	LinkedList_Clear(&record->rects, free);
+	LinkedList_Clear(&record->flash_rects, free);
 	free(record);
 }
 
-static void DrawBorder(LCUI_PaintContext paint)
+static size_t LCUIDisplay_UpdateFlashRects(SurfaceRecord record)
+{
+	size_t count = 0;
+	int64_t period;
+
+	LCUI_Graph mask;
+	LCUI_PaintContext paint;
+	LinkedListNode *node;
+	FlashRect flash_rect;
+
+	for (LinkedList_Each(node, &record->flash_rects)) {
+		flash_rect = node->data;
+		period = LCUI_GetTimeDelta(flash_rect->paint_time);
+		if (period >= FLASH_DURATION) {
+			LinkedList_DeleteNode(&record->flash_rects, node);
+			continue;
+		}
+
+		Graph_Init(&mask);
+		mask.color_type = LCUI_COLOR_TYPE_ARGB;
+		Graph_Create(&mask, flash_rect->rect.width,
+			     flash_rect->rect.height);
+		Graph_FillRect(&mask, ARGB(255, 0, 0, 200), NULL, TRUE);
+
```

Comment:
The border should be drawn for the flashing rectangle

Reply:
border should rendered with constant opacity?

Reply:
![out](https://user-images.githubusercontent.com/4847494/71320431-392e2b00-24bc-11ea-935b-0a2409bb50d6.gif)